common/crypto/ledger/ledger_secp256k1.go

```diff
@@ -0,0 +1,98 @@
+package ledger
+
+import (
+	"github.com/binance-chain/go-sdk/common/crypto"
+	"github.com/binance-chain/go-sdk/common/crypto/secp256k1"
+	"github.com/binance-chain/go-sdk/types"
+	"github.com/btcsuite/btcd/btcec"
+	ledgergo "github.com/zondax/ledger-cosmos-go"
+)
+
+var (
+	// discoverLedger defines a function to be invoked at runtime for discovering
+	// a connected Ledger device.
+	DiscoverLedger discoverLedgerFn
+)
+
+type (
+
+	// discoverLedgerFn defines a Ledger discovery function that returns a
+	// connected device or an error upon failure. Its allows a method to avoid CGO
+	// dependencies when Ledger support is potentially not enabled.
+	discoverLedgerFn func() (LedgerSECP256K1, error)
+
+	// DerivationPath represents a Ledger derivation path.
+	DerivationPath []uint32
+
+	// LedgerSECP256K1 reflects an interface a Ledger API must implement for
+	// the SECP256K1 scheme.
+	LedgerSECP256K1 interface {
```

Comment:
why not "Secp256k1" instead of "SECP256K1"

Reply:
Good suggestion.